src/preferencesControl.ts

```diff
@@ -75,7 +75,7 @@ export class PreferencesControl {
         for (let cookieCategory of this.cookieCategories) {
             if (cookieCategory.isUnswitchable) {
                 let item = `
-                <li class="${ styles.cookieListItem } ${ styles.textColorTheme }">
+                <li class="${ styles.cookieListItem } ${ styles.textColorTheme }" aria-label="${ HtmlTools.escapeHtml(cookieCategory.name) }">
```

Comment:
Let's change from `<ol><li>` to `<dl><dt>`. Narrator pronounce too much unnecessary noise with `ol`